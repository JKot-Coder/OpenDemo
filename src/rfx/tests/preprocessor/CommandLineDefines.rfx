// :RUN_RFXC -P %STD_OUTPUT% %INPUT% -D TEST -D FOO=BAR -D OK=

#ifdef PREPROCESSOR
#error 'PREPROCESSOR' should not be defined!
#endif

#ifndef TEST
#error 'TEST' must be defined!
#endif

FOO

#undef FOO
#define FOO BAR2000

FOO