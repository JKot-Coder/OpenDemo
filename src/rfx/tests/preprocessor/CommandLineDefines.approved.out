exit_status:0
output:
Source output:
    #line 11 "../src/rfx/tests/preprocessor/CommandLineDefines.rfx"
    BAR
    #line 16
    BAR2000
Diagnostic output:
